pkg/scheduler/internal/queue/scheduling_queue.go

```diff
@@ -59,7 +59,13 @@ const unschedulableQTimeInterval = 60 * time.Second
 type SchedulingQueue interface {
 	Add(pod *v1.Pod) error
 	AddIfNotPresent(pod *v1.Pod) error
-	AddUnschedulableIfNotPresent(pod *v1.Pod) error
+	// AddUnschedulableIfNotPresent adds an unschedulable pod back to the
+	// queue. Note that current scheduling cycle should be passed to decide
+	// whether we should move it to active queue instead.
+	AddUnschedulableIfNotPresent(pod *v1.Pod, podSchedulingCycle int64) error
+	// SchedulingCycle returns current scheduling cycle which should be passed
+	// with current pod to AddUnschedulableIfNotPresent.
```

Comment:
ditto, let's make the comment straightforward and give tips for developers. My stupid example is like:

```
// SchedulingCycle returns the current number of scheduling cycle which is cached by scheduling queue. 
// Normally, incrementing this number whenever a pod is popped (e.g. called Pop()) is enough.
```

Reply:
Updated. I think this is good enough. PTAL.